src/main/resources/db/migration/V1_117__update_print_files_to_clob_procedure.sql

```diff
@@ -0,0 +1,168 @@
+-- DROP PROCEDURE juror_mod.printfiles_to_clob(int4);
+
+CREATE OR REPLACE PROCEDURE juror_mod.printfiles_to_clob(IN p_limit integer DEFAULT 1000000)
+ LANGUAGE plpgsql
+AS $procedure$
+DECLARE
+	v_form_type VARCHAR(6);
+	v_max_rec_len INTEGER;
+	v_ext_date DATE;
+	forms RECORD;
+	v_count INTEGER:=0;
+
+BEGIN
+	v_ext_date :=	CASE
+						WHEN TO_CHAR(NOW(),'SSSSS')::INTEGER < 64800
```

Comment:
A comment here would be useful to explain that 64,800 seconds past midnight translates to 18:00 - so we have conditional logic to run for today if running after 6pm or run for yesterday if running before 6pm